eng/pipelines/runtimelab.yml

```diff
@@ -49,19 +53,36 @@ stages:
     parameters:
       osGroup: Windows_NT
       archType: x64
-      pool:
-        vmImage: 'windows-latest'
+      ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+        isOfficialBuild: true
+        runTests: false
+        pool:
+          name: NetCoreInternal-Pool
+          queue: BuildPool.Windows.10.Amd64.VS2019
+      ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+        pool:
+          vmImage: 'windows-latest'
 
-  - template: /eng/pipelines/templates/build-job.yml
-    parameters:
-      osGroup: OSX
-      archType: x64
-      pool:
-        vmImage: 'macOS-latest'
+  - ${{ if or(eq(variables['System.TeamProject'], 'public'), in(variables['Build.Reason'], 'PullRequest')) }}:
+    - template: /eng/pipelines/templates/build-job.yml
+      parameters:
+        osGroup: OSX
+        archType: x64
+        pool:
+          vmImage: 'macOS-latest'
 
-  - template: /eng/pipelines/templates/build-job.yml
+    - template: /eng/pipelines/templates/build-job.yml
+      parameters:
+        osGroup: Linux
+        archType: x64
+        pool:
+          vmImage: 'ubuntu-latest'
+
+# Publish and validation steps. Only run in official builds
+- ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+  - template: \eng\common\templates\post-build\post-build.yml
     parameters:
-      osGroup: Linux
-      archType: x64
-      pool:
-        vmImage: 'ubuntu-latest'
+      publishingInfraVersion: 3
+      validateDependsOn:
+        - build
+      enableSourceLinkValidation: true
```

Comment:
should we also enable validation/publishing of symbols or does that happen automatically?

Reply:
publishing of symbols is already happening. Validation I didn't enable it because I remember there was problems with it due to a delay in the symbol server when you publish new symbols. @riarenas @epananth is that still the case?

Reply:
@safern yes, I believe that's still the case.